.github/workflows/rerun_workflow_on_merge.yml

```diff
@@ -0,0 +1,28 @@
+name: Rerun Workflows on PR Merge
+
+on:
+  pull_request:
+    types:
+      - closed
+
+jobs:
+  rerun-workflows:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Trigger workflow runs
+        run: |
+          token="${{ secrets.GITHUB_TOKEN }}"
```

Comment:
I think we should mask this so that it doesn't potentially leak in the logs

Reply:
done